docusaurus/docs/cloud/projects/overview.md

```diff
@@ -46,8 +44,6 @@ From this tab you can:
 * Quickly view project [**Usage**](./setting#usage)
```

Comment:
Please change the link in line 46 to `/cloud/projects/settings#usage`, otherwise the build won't pass :) 
![Screenshot 2023-03-20 at 17 28 23](https://user-images.githubusercontent.com/4233866/226406182-fe130d8e-a605-4b4f-aeb0-f062259b55e3.png)

Reply:
Thank you, I fixed the broken link.